src/main/java/edu/wisc/my/portlets/feedback/web/FeedbackFormController.java

```diff
@@ -96,6 +110,9 @@ protected Object formBackingObject(PortletRequest request) throws Exception {
         final String telephoneNumber = userInfo.get("telephoneNumber");
         feedback.setPhoneNumber(telephoneNumber);
         feedback.setHiddenPhoneNumber(telephoneNumber);
+        feedback.setChatLink(this.preferences.getValue("chatLink", null));
+        feedback.setCallLink(this.preferences.getValue("callLink", null));
+        feedback.setHowToLink((this.preferences.getValue("howToLink", null)));
```

Comment:
I would say, it fails gracefully. Better to have a feedback form with no links than an error.